test/e2e/daemon_restart.go

```diff
@@ -241,6 +241,9 @@ var _ = Describe("DaemonRestart", func() {
 
 	It("Scheduler should continue assigning pods to nodes across restart", func() {
 
+		//TODO: check why this test doesn't work on GKE.
```

Comment:
So to be clear - I would write:
"// TODO: Enebale this test in GKE once experimental API there is switched on"
(or sth like that).